public/components/overview/mitre_attack_intelligence/resource_detail_references_table.tsx

```diff
@@ -21,32 +21,44 @@ import {
 } from '@elastic/eui';
 import { Markdown } from '../../common/util/markdown';
 
-export const GetFlyoutSubtable = ({subtableName, subtableArray, subtableProps}) => {
+type tablePropsType = (item) => {onClick: () => void};
+type backToTopType = () => void;
+
+interface referencesTableType {
+  referencesName: string,
+  referencesArray: Array<string>,
+  tableProps: tablePropsType,
+  backToTop: backToTopType
+}
+
+export const ReferencesTable = ({referencesName, referencesArray, tableProps, backToTop} : referencesTableType) => {
   const [isLoading, setIsLoading] = useState(true);
```

Comment:
have you forgotten this `const [isLoading, setIsLoading] = useState<boolean>(true);`